lib/dealership.rb

```diff
@@ -0,0 +1,45 @@
+class Dealership
+  attr_reader :name,
+              :address,
+              :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.size
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+     @inventory.size > 1
+  end
+
+  def cars_by_make(make)
+    @inventory.find_all do |car|
+      car.make == make
+    end
+  end
+
+  def total_value
+    @inventory.sum do |car|
+      car.total_cost
+    end
+  end
+
+  def details
+    {
+      "total_value" => total_value,
+      "address" => @address
+    }
+  end
+
+  def average_price_of_car
+    ((total_value / @inventory.size).to_s).insert(2, ",")
```

Comment:
Nice job here. You could have also divided `total_value` by your `inventory_count` method to keep your code DRY. 

You also did not write any tests for this.